environments/testing-state.tf

```diff
@@ -0,0 +1 @@
+
```

Comment:
This file should be dropped, we shouldn't need to commit this?

Reply:
File can't be dropped since acp-toolset relies on it to run it's testing. [Here's a link to the issues](https://github.com/UKHomeOffice/acp-toolset/issues/29)